docs_src/examples/Ch-OSImageWithEdgeX.md

```diff
@@ -99,71 +71,74 @@ Find your developer ID using the Snapcraft CLI:
 ```bash title="🖥 Desktop"
 $ snapcraft whoami
 ...
-developer-id: SZ4OfFv8DVM9om64iYrgojDLgbzI0eiL
+developer-id: <developer-id>
 ```
 or from the [Snapcraft Dashboard](https://dashboard.snapcraft.io/dev/account/).
 
-!!! info
+??? info "YAML Model Assertion"
     Unlike the official documentation which uses JSON, we use YAML serialization for the model. This is for consistency with all the other serialization formats in this tutorial. Moreover, it allows us to comment out some parts for testing or add comments to describe the details inline.
 
 Create `model.yaml` with the following content, replacing `authority-id`, `brand-id`, and `timestamp`:
 ```yaml
 type: model
 series: '16'
 
-# authority-id and brand-id must be set to your developer-id
-authority-id: SZ4OfFv8DVM9om64iYrgojDLgbzI0eiL
-brand-id: SZ4OfFv8DVM9om64iYrgojDLgbzI0eiL
+# set authority-id and brand-id your developer-id
+authority-id: <developer-id>
+brand-id: <developer-id>
 
-model: ubuntu-core-20-amd64
+model: ubuntu-core-22-amd64
 architecture: amd64
 
 # timestamp should be within your signature's validity period
 timestamp: '2022-06-21T10:45:00+00:00'
```

Comment:
Is it possible to include instructions on how to determine the validity period of a signature?
```suggestion
timestamp: '2023-06-21T10:45:00+00:00'
```
In my case, using the default timestamps results in the following error:
```
$ ubuntu-image snap model.signed.yaml --validation=enforce
Error: Error preparing image: cannot fetch and check prerequisites for the model assertion: cannot add assertion model (ubuntu-core-22-amd64; series:16 brand-id:3ehWphH4KXEI2BY8mla8ydYPJm8BCfkd): model assertion timestamp "2022-06-21 10:45:00 +0000 +0000" outside of signing key validity (key valid since "2023-03-07 13:18:43 +0000 UTC")
```

Reply:
> Is it possible to include instructions on how to determine the validity period of a signature?

The error message shows the signature's validity.